src/it/scala/temple/containers/PostgresSpec.scala

```diff
@@ -31,4 +32,23 @@ trait PostgresSpec extends FlatSpec with DockerTestKit with DockerPostgresServic
 
   // Execute a query that does not return a result
   def executeWithoutResults(query: String): Unit = execute { _.createStatement().execute(query) }
+
+  def executeWithoutResultsPrepared(preparedStatement: String, values: List[PreparedVariable]): Unit = execute { con =>
```

Comment:
Just style: connection is abbreviated as `conn` above in `execute`, but as `con` here.